tests/test_timeseriesflattener/test_resolve_multiple.py

```diff
@@ -298,3 +298,26 @@ def test_resolve_multiple_change_per_day_too_few_datapoints():
         expected_flattened_values=[1, 99999],
         fallback=99999,
     )
+
+
+def test_resolve_multiple_mean_of_multiple_columns():
+    prediction_times_str = """dw_ek_borger,timestamp,
+                            1,2021-12-31 00:00:00
+                            2,2021-12-31 00:00:00
+                            """
+    event_times_str = """dw_ek_borger,timestamp,val1,val2
+                        1,2022-01-01 00:00:00, 1,2
+                        1,2022-01-02 00:00:00, 2,3
+                        2,2022-01-01 00:00:00, 3,4
+                        2,2022-01-08 00:00:00, 4,5
+                        """
+
+    assert_flattened_outcome_as_expected(
+        prediction_times_df_str=prediction_times_str,
+        outcome_df_str=event_times_str,
+        resolve_multiple="mean",
+        lookahead_days=4,
+        expected_flattened_values=[[1.5, 2.5], [3.0, 4.0]],
```

Comment:
```suggestion
        expected_flattened_values=[[1.5, 2.5], [3.0, 4.0]],
```

This is hard to read for me – e.g. does 3.0 map to `dw_ek_borger` 1 or 2? 

I.e. it's not obvious for me if the two lists represent the two columns or the two citizens. I can infer it, but only because I know what the results are supposed to be.

Would love if it could take a dict instead, where the keys are the colnames and the vals are the expected values.

Reply:
Agreed, pandas list handling is a bit hard to read.
Don't think it's worth rewriting the function - would make this slightly easier to read at the expense of making the next function significantly harder.